_episodes/02-numpy.md

```diff
@@ -546,6 +546,20 @@ which is the average inflammation per patient across all days.
 > > Creates a substring from index 1 up to (not including) the final index,
 > > effectively removing the first and last letters from 'oxygen'
 > {: .solution}
+>
+> How can we rewrite the slice for getting the last three characters of `element`,
+> so it will work on any character string at least three characters in length?
```

Comment:
Lauren, what do you think about this:

> How can we rewrite the slice for getting the last three characters of `element` so that it works even if we assign a different string to `element`? Test your solution with the following strings: `carpentry`, `clone`, `hi`.

Reply:
I think that change is fine to make. It might be nice having `hi` as an example, since it will show learners that it doesn't error to take a slice bigger than the string.